smithy-aws-protocol-tests/model/awsJson1_0/defaults.smithy

```diff
@@ -0,0 +1,357 @@
+$version: "2.0"
+
+namespace aws.protocoltests.json10
+
+use aws.protocols#awsJson1_0
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+apply OperationWithDefaults @httpRequestTests([
+    {
+        id: "AwsJson10ClientPopulatesDefaultValuesInRequestBodyIfMissingInParams",
+        documentation: "Client populates default values in request body if missing in params.",
+        protocol: awsJson1_0,
+        method: "POST",
+        bodyMediaType: "application/json",
+        uri: "/",
+        headers: {
+            "Content-Type": "application/x-amz-json-1.0"
+        },
+        body: """
```

Comment:
Clients ignore defaults on `@input` shapes. It's treated as implicitly clientOptional based on the spec. So to test that clients serialize defaults, you should add a nested structure.